content: Derelict ship encounters (#9039)
endless-sky/endless-sky: data/human/human missions.txt

				clear "Blind Man from Martini: label looking"
			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`

mission "Derelict Informer"
	minor
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		not attributes "uninhabited"
		not system "Rutilicus"
	to offer
		random < 4
	on offer
		conversation
			`As you touch down on <origin>, you see that the landing pad next to yours holds a freighter that is practically falling apart. Standing next to it is a man with the grizzled look of an experienced captain.`
			`	Given the sorry state of the ship he might be in need of assistance.`
			choice
				`	(Offer your help.)`
				`	(Don't help him.)`
					decline
			`	The captain notices you approaching, and clearly guessing your intentions, speaks. "Oh don't worry, this ain't my ship."`
			choice
				`	"What are you doing with it then?"`
					goto what
				`	"Whose is it then?"`
			`	"Well quite simply, it's nobody's."`
			choice

> 				`	"What are you doing with it then?"`

Reviewer: I'd add another dialog option here for variety. It doesn't need to lead to another label, but just something for roleplay purposes.

			choice
				`	(Loot the treasure.)`
					goto end
			label end
			action
				payment 386000
				outfit "Ion Cannon" 2
			`	You grab the <payment> credits and with the assistance of some machinery you bring the weapons back to your ship.`

mission "Derelict Family Investment 1"
	description "Return the credits you found in a derelict ship to the family it belongs to."
	name "Colin Sawyer's Investment"
	landing
	to offer
		has "derelict family investment"
	destination "Sundive"
	on offer
		conversation
			`Upon landing on <origin> you take out the text log you recovered alongside the case of credits from the derelict ship. The log reveals the credits were the result of a risky investment in an attempt to buy a better life for the captain of the ship, Colin Sawyer, and his family. You assume that the captain never made it home, and looking further into the personal log you find that the family used to reside on <destination>.`
			`	Perhaps the family still live there?`
			choice
				`	(Attempt to return the credits.)`
					accept
				`	(Keep the credits to yourself.)`
			action
				payment 325000

> 			`	You decide to keep the credits to yourself. After all, it's not every day that you find a score worth <payment> credits.`

Reviewer: ```suggestion
			`	You decide to keep the money. After all, it's not every day that you find a score worth <payment>.`
```

				decline
	on complete
		conversation
			`	Upon landing on <planet> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`
			choice
				`	"Do you know a Colin Sawyer?"`
				`	"Do the Sawyers live here?"`
			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyer's new place of residence and bids you farewell.`

mission "Derelict Family Investment 2"
	description "Return the credits you found in a derelict ship to the family it belongs to."
	name "Colin Sawyer's Investment"
	landing
	to offer
		has "Derelict Family Investment 1: done"
	destination "Foundry"
	on complete
		payment 500000
		conversation

> 			`	It doesn't take long to find the Sawyers' home on <planet>. The building makes it clear that the family has done well for themselves. After a quick explanation with the desk clerk that you have something important to give, you take an elevator ride to the penthouse. You ring the buzzer and are greeted by a woman in her late fifties, who asks, "A delivery?"`

Reviewer: ```suggestion
			`	It doesn't take long to find the Sawyers' home on <planet>. The building makes it clear that the family has done well for themselves. After a quick explanation with the desk clerk that you have something important to give, you take an elevator ride to the penthouse. You ring the buzzer and are greeted by a woman in her late fifties. "A delivery?" she asks.`
```

		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`

mission "Derelict Treasure Map: Ion Cannons"
	landing
	name "Derelict Treasure Map"
	description "Follow the treasure map you found in a derelict ship to <destination>."
	destination "New Argentina"
	to offer
		has "derelict treasure map: ion cannons"

> 	on offer

Reviewer 2: ```suggestion
```

	on offer
		conversation
			`Upon landing on <origin> you take out the treasure map you recovered from the derelict ship. The map gives detailed instructions about where the treasure was buried on the sparsely populated world of <destination>. However, the map's creator claims to have obtained some kind of alien weapons that disrupts a ship's electrical systems. The outlandish claim seems to bring doubt to the validity of the map.`
			choice
				`	(Follow the map.)`
					accept
				`	(Throw the map away, it's probably fake.)`
					decline

> 	on complete

Reviewer 2: The conversation action at the end of this convo could just move its actions to the on complete, as you'll hit the conversation action regardless of what choice you make.
```suggestion
	on complete
		payment 386000
		outfit "Ion Cannon" 2
```

		conversation
			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shoveling the hard soil you're met with a thunk as you strike metal.`
			branch hai
				has "First Contact: Hai: offered"
			`	Opening the cache reveals a large store of credits along with the alien weapons the map's owner wrote about. You marvel at their strange design and wonder at their power.`
			choice
				`	(Loot the treasure.)`
					goto end
			label hai
			`	Opening the cache reveals a large store of credits along with two Hai Ion Cannons. These must be the alien weapons the map's owner wrote about.`
			choice
				`	(Loot the treasure.)`
					goto end
			label end

> 			action
> 				payment 386000
> 				outfit "Ion Cannon" 2

Reviewer 2: ```suggestion
```

			`	You grab the <payment> credits, and with the assistance of some machinery you bring the weapons back to your ship.`

mission "Derelict Family Investment 1"

> 	name "Lost Family Investment"
> 	description "Return the credits you found in a derelict ship to the family it belongs to."
> 	landing
> 	to offer
> 		has "derelict family investment"
> 	destination "Sundive"

Reviewer 2: ```suggestion
	landing
	name "Lost Family Investment"
	description "Return the credits you found in a derelict ship to the family it belongs to by traveling to <destination>."
	destination "Sundive"
	to offer
		has "derelict family investment"
```

> 	on offer

Reviewer 2: I don't think the first payment replacement in this conversation is going to work because there is no on offer or on complete that contains a payment. The second payment replacement will work because the action containing the payment is directly above it. Therefore, we could use an apparent payment here to fix the first replacement.
```suggestion
	"apparent payment" 325000
	on offer
```

		conversation
			`Upon landing on <origin> you take out the text log you recovered alongside the case of <payment> from the derelict ship. The log reveals the credits were the result of a risky investment in an attempt to buy a better life for the captain of the ship, Colin Sawyer, and his family. You assume that the captain never made it home, and looking further into the personal log you find that the family used to reside on <destination>.`
			`	Perhaps the family still lives there.`
			choice
				`	(Attempt to return the credits.)`
					accept
				`	(Keep the credits to myself.)`
			action
				payment 325000
			`	You decide to keep the money. After all, it's not every day that you find a score worth <payment>.`
				decline

mission "Derelict Family Investment 2"

> 	name "Lost Family Investment"
> 	description "Return the credits you found in a derelict ship to the family it belongs to."
> 	landing
> 	to offer
> 		has "Derelict Family Investment 1: done"
> 	destination "Foundry"

Reviewer 2: ```suggestion
	landing
	name "Lost Family Investment"
	description "Return the credits you found in a derelict ship to the family it belongs to by traveling to <destination>."
	destination "Foundry"
	to offer
		has "Derelict Family Investment 1: done"
```

	on offer
		conversation

> 			`Upon landing on <planet> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`

Reviewer 2: ```suggestion
			`Upon landing on <origin> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`
```

			choice
				`	"Do you know a Colin Sawyer?"`
				`	"Do the Sawyers live here?"`

> 			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyers' new place of residence on <destination> and bids you farewell.`

Reviewer 2: ```suggestion
			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyers' new place of residence on <destination> and bids you farewell.`
				accept
			
```

	on complete
		payment 500000
		conversation

> 			`It doesn't take long to find the Sawyers' home on <planet>. The building makes it clear that the family has done well for themselves. After a quick explanation with the desk clerk that you have something important to give, you take an elevator ride to the penthouse. You ring the buzzer and are greeted by a woman in her late fifties. "A delivery?" she asks.`

Reviewer 2: ```suggestion
			`It doesn't take long to find the Sawyers' home on <planet>. The building makes it clear that the family has done well for themselves. After a quick explanation to the desk clerk that you have something important to give, you take an elevator ride to the penthouse. You ring the buzzer and are greeted by a woman in her late fifties. "A delivery?" she asks.`
```

		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`

> mission "Derelict Informer"

Reviewer 2: Honestly not so sure on including this mission. It feels very game-y, like it's only purpose is to inform the player about a game mechanic rather than it being an organic encounter. Why not leave derelict ships as something for the player to discover on their own instead of having someone tell you about them?

Author: Initially I planned on having the derelicts only in the 2 jump uninhabited systems and would have been difficult to find since these systems are pretty much avoided once discovered. I've since included them in the 1 jump systems which are commonly passed through, so perhaps the mission could be excluded, or expanded to be a proper paying mission while still teaching about derelicts.

Reviewer 2: I think I'd prefer to just leave this mission out.

	minor
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		not attributes "uninhabited"

> 		not system "Rutilicus"
> 	to offer
> 		random < 4

Reviewer 2: Making sure the date is X number of days after the start of the game would make more sense than explicitly excluding New Boston, as there may be other starts at other locations in the future that would need added to this location filter.
```suggestion
	to offer
		random < 4
		"days since start" > 30
```

> 		"days since start" > 30

Author: Actually Intro 0 doesn't seem to be priority like the rest? Which is the one that would affect this, that seems like a separate issue that needs to be fixed though.

> 		"days since start" > 30

Reviewer 2: Ah, yeah, because the first intro mission is actually a landing mission (which can't have priority). This mission offers in the spaceport, which means the intro mission being a landing mission won't block it.

	on offer
		conversation

> 			`As you touch down on <origin>, you see that the landing pad next to yours holds a freighter that is practically falling apart. Standing next to it is a man with the grizzled look of an experienced captain.`

Reviewer 2: Is this a freighter or is this a Freighter?

Author: generic freighter seems fine

			`	Given the sorry state of his ship he might be in need of assistance.`
			choice
				`	(Offer your help.)`
				`	(Don't help him.)`
					decline
			`	The captain notices you approaching, and clearly guessing your intentions, speaks. "Oh don't worry, this ain't my ship."`
			choice
				`	"What are you doing with it then?"`
					goto what
				`	"Whose is it then?"`
			`	"Well quite simply, it's nobody's."`
			choice
				`	"What are you doing with it then?"`
				`	"If it isn't yours, why do you have it?"`
			label what
			`	An easy, broad smile upon his face tells you he's heard such a question before. "Glad you asked. It's always a pleasure to teach another captain about the niche trade of derelict hunting."`
			choice
				`	"Derelict hunting?"`

> 					goto confused

Reviewer 2: ```suggestion
```

				`	"So you search for abandoned ships?"`
					goto precisely

> 			label confused

Reviewer 2: ```suggestion
```

> 			`	"Derelict hunting involves searching abandoned ship hulls. They usually have a few parts left on them that you can sell for a few credits, and if you're lucky you might even score some rare outfits or the whole hull if it's stable enough. Wouldn't recommend actually using that hull though. They're usually very fragile like this freighter here."`

Reviewer 2: All of the derelict variants are uncapturable. Not sure how much it makes sense to say that you can capture derelict ships.

			choice
				`	"So you search for abandoned ships and loot them?"`
				`	"I get it, you're recycling old ships for profit."`
			label precisely
			`	"Precisely. If you want to go hunting for derelicts yourself, they can be rarely found in out of the way uninhabited systems. Good hunting out there." He turns back to his score.`
				decline

> mission "Derelict Treasure Map"
> 	description "Follow the treasure map you found in a derelict ship."
> 	landing
> 	to offer
> 		has "derelict treasure map"
> 	destination "New Argentina"

Reviewer 2: We might have other treasure map missions in the future, so I'd give this a `name` node and change the identifier.
```suggestion
mission "Derelict Treasure Map: Ion Cannons"
	landing
	name "Derelict Treasure Map"
	description "Follow the treasure map you found in a derelict ship to <destination>."
	destination "New Argentina"
	to offer
		has "derelict treasure map"
	on offer
```

	on offer
		conversation
			`Upon landing on <origin> you take out the treasure map you recovered from the derelict ship. The map gives detailed instructions about where the treasure was buried on the sparsely populated world of <destination>. However, the map's creator claims to have obtained some kind of alien weapons that disrupts a ship's electrical systems. The outlandish claim seems to bring doubt to the validity of the map.`
			choice
				`	(Follow the map.)`
					accept
				`	(Throw the map away, it's probably fake.)`
					decline
	on complete
		conversation

> 			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shovelling the hard soil you're met with a thunk as you strike metal.`

Reviewer 2: ```suggestion
			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shovelings the hard soil you're met with a thunk as you strike metal.`
```

Author: I assume you just meant to drop an l and not add the s?

Reviewer 2: Yeah.

			branch hai
				has "First Contact: Hai: offered"
			`	Opening the cache reveals a large store of credits along with the alien weapons the map's owner wrote about. You marvel at their strange design and wonder at their power.`
			choice
				`	(Loot the treasure.)`
					goto end
			label hai
			`	Opening the cache reveals a large store of credits along with two Hai Ion Cannons. These must be the alien weapons the map's owner wrote about.`
			choice
				`	(Loot the treasure.)`
					goto end
			label end
			action
				payment 386000
				outfit "Ion Cannon" 2
			`	You grab the <payment> credits, and with the assistance of some machinery you bring the weapons back to your ship.`

mission "Derelict Family Investment 1"

> 	name "Lost Family Investment"
> 	description "Return the credits you found in a derelict ship to the family it belongs to."
> 	name "Colin Sawyer's Investment"
> 	landing
> 	to offer
> 		has "derelict family investment"
> 	destination "Sundive"

Reviewer 2: You have two name nodes. Which is it?
```suggestion
	landing
	name "Lost Family Investment"
	name "Colin Sawyer's Investment"
	description "Return the credits you found in a derelict ship to the family it belongs to."
	destination "Sundive"
	to offer
		has "derelict family investment"
```

	on offer
		conversation

> 			`Upon landing on <origin> you take out the text log you recovered alongside the case of credits from the derelict ship. The log reveals the credits were the result of a risky investment in an attempt to buy a better life for the captain of the ship, Colin Sawyer, and his family. You assume that the captain never made it home, and looking further into the personal log you find that the family used to reside on <destination>.`

Reviewer 2: Again, say how many credits it is. We can't really make a decision if we don't know what we're dealing with.

> 			`	Perhaps the family still live there?`

Reviewer 2: ```suggestion
			`	Perhaps the family still live there.`
```

			choice
				`	(Attempt to return the credits.)`
					accept

> 				`	(Keep the credits to yourself.)`

Reviewer 2: ```suggestion
				`	(Keep the credits to myself.)`
```

			action
				payment 325000
			`	You decide to keep the money. After all, it's not every day that you find a score worth <payment>.`
				decline
	on complete
		conversation

> 			`	Upon landing on <planet> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`

Reviewer 2: ```suggestion
			`Upon landing on <planet> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`
```

			choice
				`	"Do you know a Colin Sawyer?"`
				`	"Do the Sawyers live here?"`

> 			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyer's new place of residence and bids you farewell.`

Reviewer 2: This should be a part of the on offer conversation of the next mission. That way you can actually tell the player what the destination is with a text replacement.

mission "Derelict Family Investment 2"

> 	name "Lost Family Investment"
> 	description "Return the credits you found in a derelict ship to the family it belongs to."
> 	name "Colin Sawyer's Investment"
> 	landing
> 	to offer
> 		has "Derelict Family Investment 1: done"
> 	destination "Foundry"

Reviewer 2: Same here.
```suggestion
	landing
	name "Lost Family Investment"
	name "Colin Sawyer's Investment"
	description "Return the credits you found in a derelict ship to the family it belongs to."
	destination "Foundry"
	to offer
		has "Derelict Family Investment 1: done"
```

	on complete
		payment 500000
		conversation

> 			`	It doesn't take long to find the Sawyers' home on <planet>. The building makes it clear that the family has done well for themselves. After a quick explanation with the desk clerk that you have something important to give, you take an elevator ride to the penthouse. You ring the buzzer and are greeted by a woman in her late fifties. "A delivery?" she asks.`

Reviewer 2: ```suggestion
			`It doesn't take long to find the Sawyers' home on <planet>. The building makes it clear that the family has done well for themselves. After a quick explanation with the desk clerk that you have something important to give, you take an elevator ride to the penthouse. You ring the buzzer and are greeted by a woman in her late fifties. "A delivery?" she asks.`
```

			choice
				`	(Loot the treasure.)`
					goto end
			label hai
			`	Opening the cache reveals a large store of credits along with two Hai Ion Cannons. These must be the alien weapons the map's owner wrote about.`
			choice
				`	(Loot the treasure.)`
					goto end
			label end
			action
				payment 386000
				outfit "Ion Cannon" 2
			`	You grab the <payment> credits, and with the assistance of some machinery you bring the weapons back to your ship.`

mission "Derelict Family Investment 1"
	name "Lost Family Investment"
	description "Return the credits you found in a derelict ship to the family it belongs to."
	landing
	to offer
		has "derelict family investment"
	destination "Sundive"
	on offer
		conversation
			`Upon landing on <origin> you take out the text log you recovered alongside the case of <payment> from the derelict ship. The log reveals the credits were the result of a risky investment in an attempt to buy a better life for the captain of the ship, Colin Sawyer, and his family. You assume that the captain never made it home, and looking further into the personal log you find that the family used to reside on <destination>.`

> 			`	Perhaps the family still live there.`

Reviewer 3: ```suggestion
			`	Perhaps the family still lives there.`
```

			choice
				`	(Attempt to return the credits.)`
					accept
				`	(Keep the credits to myself.)`
			action
				payment 325000
			`	You decide to keep the money. After all, it's not every day that you find a score worth <payment>.`
				decline

mission "Derelict Family Investment 2"
	name "Lost Family Investment"
	description "Return the credits you found in a derelict ship to the family it belongs to."
	landing
	to offer
		has "Derelict Family Investment 1: done"
	destination "Foundry"
	on offer
		conversation
			`Upon landing on <planet> you take a transport out to the small farm indicated in the log as the home of Captain Sawyer's family. After knocking on the door of the farmhouse a man opens the door and asks, "Can I help you?"`
			choice
				`	"Do you know a Colin Sawyer?"`
				`	"Do the Sawyers live here?"`

> 			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyer's new place of residence on <destination> and bids you farewell.`

Reviewer 3: ```suggestion
			`	"Sawyer? Yeah, that's the family that sold me this here farm. If you're looking for them I believe they moved to a more populated planet, own some big corporation now." The man gives you the Sawyers' new place of residence on <destination> and bids you farewell.`
```

	on complete

> 		payment 500000

Author: No, she lets you keep the credits and gives you extra for returning the log.

> 		payment 500000

Reviewer 4: Ah, OK.
